darwin/exporter/formats/nifti.py

```diff
@@ -136,15 +134,21 @@ def export_single_nifti_file(video_annotation: dt.AnnotationFile, output_dir: Pa
             dataobj=np.flip(output_volumes[class_name], (0, 1, 2)).astype(np.int16),
             affine=affine,
         )
+        if original_affine is not None:
+            img_ornt = io_orientation(original_affine)  # Get orientation of current affine
+            ras_ornt = axcodes2ornt("RAI")  # Get RAI orientation
+            from_canonical = ornt_transform(ras_ornt, img_ornt)  # Get transform from RAS to current affine
+            img = img.as_reoriented(from_canonical)
         output_path = Path(output_dir) / f"{image_id}_{class_name}.nii.gz"
         if not output_path.parent.exists():
             output_path.parent.mkdir(parents=True)
         nib.save(img=img, filename=output_path)
 
 
-def shift_polygon_coords(polygon, height, width, pixdim):
+def shift_polygon_coords(polygon):
     # Need to make it clear that we flip x/y because we need to take the transpose later.
-    return [{"x": p["y"] * float(pixdim[0]), "y": p["x"] * float(pixdim[1])} for p in polygon]
+    # TODO: We might need to add a correction factor here for anisotropic pixdims.
```

Comment:
The fix that was in there was wrong. Anisotropy in the z-axis is handled automatically, given the pixel array and the affine. The issue is anisotropy in the in-plane will crop up, but I want to cross that bridge when we get to it and push this along. Will be an easy update to roll out when the time comes, and easier to implement when we have a client use case its failing on. Hope that's good enough for this to pass.

Reply:
I understand, was only trying to protect against unidirectional pitfalls.

So this doesn't actually cause a regression, as what was there wasn't fixed anyways. Can we make sure to ticket this case and add it to the backlog?

otherwise doesn't block this PR.